Automattic/wp-calypso
# Use package to format prices in Plan Store

#### Changes proposed in this Pull Request

* Instead of creating own formatting logic, make the plan store use the format-currency package to format prices. 

#### Testing instructions

1. Go to https://calypso.live/new?branch=update/plan-store-price-formats.
2. Make sure the plans grid looks good.

#### Note
I tested with #48963 locally and it works great. I (or Marco) will rebase #48963 once this is merged.
